Task8/src/main/java/org/andersen/controller/UserController.java

```diff
@@ -0,0 +1,45 @@
+package org.andersen.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import lombok.RequiredArgsConstructor;
+import org.andersen.exception.UserActivationIsDisabledException;
+import org.andersen.model.user.User;
+import org.andersen.service.UserService;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/user/")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/{id}")
+    @ApiOperation(value = "Get user")
+    @ApiResponse(code = 200, message = "OK")
+    public User getUser(@PathVariable("id") Long id){
+        return userService.getUserById(id);
+    }
+
+    @PostMapping("/new")
```

Comment:
We could skip here /new :)
It doesn't have any specific info - POST method itself tells us that we're going to create object